Add the draw_plotly method and a tutorial notebook

FromBei-JingWithLove opened the pull request:
draw_plotly method: visualize 3D models using plotly so that we can directly see models in cloud notebooks.

<!-- Reviewable:start -->
- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/isl-org/open3d/5418)
<!-- Reviewable:end -->

errissa reviewed (changes requested):
> Reviewed 2 of 4 files at r1, 3 of 3 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/isl-org/open3d/5418)* status: all files reviewed, 1 unresolved discussion (waiting on @FromBei-JingWithLove and @yxlao)

___
*[`python/open3d/visualization/draw.py` line 35 at r3](https://reviewable.io/reviews/isl-org/open3d/5418#-NC1Hjxw3wmErBHMyfLZ:-NC1Hjxw3wmErBHMyfL_:bmgrw7d) ([raw file](https://github.com/isl-org/open3d/blob/2d90d8246134494248aa214287e8155c7b075559/python/open3d/visualization/draw.py#L35)):*
> ```Python
> import open3d as o3d
> import numpy as np
> import plotly.graph_objects as go
> ```

Are all these import necessary? They don't get used in `draw.py`.


<!-- Sent from Reviewable.io -->

FromBei-JingWithLove commented:
> > Reviewed 2 of 4 files at r1, 3 of 3 files at r3, all commit messages.
> > _[Reviewable](https://reviewable.io/reviews/isl-org/open3d/5418)_ status: all files reviewed, 1 unresolved discussion (waiting on @FromBei-JingWithLove and @yxlao)
> 
> _[`python/open3d/visualization/draw.py` line 35 at r3](https://reviewable.io/reviews/isl-org/open3d/5418#-NC1Hjxw3wmErBHMyfLZ:-NC1Hjxw3wmErBHMyfL_:bmgrw7d) ([raw file](https://github.com/isl-org/open3d/blob/2d90d8246134494248aa214287e8155c7b075559/python/open3d/visualization/draw.py#L35)):_
> 
> > ```python
> > import open3d as o3d
> > import numpy as np
> > import plotly.graph_objects as go
> > ```
> 
> Are all these import necessary? They don't get used in `draw.py`.

No. I added the draw_plotly method in draw.py so I added the imports. I will delete them and make a new commit.

errissa reviewed (approved):
> Reviewed 1 of 1 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/isl-org/open3d/5418)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @yxlao)



<!-- Sent from Reviewable.io -->
